tests/fsyacc/OldFsYaccTests.fsx

```diff
@@ -161,6 +163,17 @@ runTests test1unicodeProj [
     test1Input1, test1Input1Bsl
     sprintf "%s %s" test1Input2Variation1 test1Input2Variation2, test1Input2Bsl
     sprintf "--tokens %s" test1unicodeInput3, test1unicodeInput3TokensBsl
+    //sprintf "--tokens %s" test1unicodeInput4, test1unicodeInput4TokensBsl
```

Comment:
I would like to transform this line into a tests that shows that the lexer without `-i` fails on this input.

Reply:
Done